jest.config.js

```diff
@@ -1,3 +1,23 @@
+/*eslint-disable */
```

Comment:
Couldn't figure this one out. Something kept adding the `;` to lines 1 and 11, but then lint would complain saying there were `;`. Would be nice to figure this one out and remove the `eslint-disable`. Could use your help here.